mdsobjects/python/mdsdata.py

```diff
@@ -386,7 +386,7 @@ def __fixTreeReferences__(self,tree):
     def decompile(self):
         """Return string representation
         @rtype: string"""
-        return _cmp.DECOMPILE(self)._setTree(self.tree).evaluate()
+        return str(_cmp.DECOMPILE(self)._setTree(self.tree).evaluate())
```

Comment:
will this work on python3 as well … should be handled by __str__ but just checking